src/sass/_base.scss

```diff
@@ -1,5 +1,117 @@
 * {
   box-sizing: border-box;
+  font-family: "Jetbrains Mono", sans-serif;
 }
 
-body {}
+body {
+  background-color: $background;
+  margin: 0;
+}
+
+section {
+  border: 1px solid $button;
+  border-radius: 5px;
+  color: $light;
+  margin: 40px 10px;
+  padding: 5px;
+  text-align: center;
+
+  h2 {
+    margin: 0 auto;
+    width: 80%;
+  }
+
+  h3 {
+    margin: 10px auto;
+    width: 80%;
+  }
+
+  p {
+    margin: 40px auto;
+    padding: 0 20%;
+    width: 90%;
+  }
+
+  a {
+    color: $light;
+  }
+}
+
+.calendar {
+  margin: 0 auto;
+  text-align: left;
+  width: 150px;
+
+  .submit {
+    background-color: $button;
+    border: none;
+    border-radius: 10px;
+    color: #fff;
+    display: block;
+    font-size: 24px;
+    font-weight: 700;
+    margin: 0 auto;
+    margin: 20px;
+    padding: 20px;
+  }
+}
+
+.site-name {
+  color: $light;
+  font-size: 36px;
+  margin: 40px auto 20px auto;
+  text-decoration: none;
+}
+
+.top {
+  font-size: 30px;
+  font-weight: 900;
+}
+
+.bot {
+  font-size: 20px;
+  font-weight: 700;
+}
+
+.title {
+  color: $light;
+  font-size: 40px;
+  margin: 20px auto;
+  width: 80%;
+}
+
+.landing img {
+  height: 400px;
+  margin: 0 auto;
+  object-fit: cover;
+  overflow: hidden;
+  width: 70%;
+}
+
+
+
+button {
+  background-color: transparent;
+  border: 0;
+  margin: 0 auto;
+  opacity: 1;
+  padding: 0;
+  width: 60%;
+
+  a {
+    background-color: $button;
+    border-radius: 10px;
+    color: #fff;
+    display: block;
+    font-size: 24px;
+    font-weight: 700;
+    margin: 20px;
+    padding: 20px;
+    text-decoration: none;
+  }
+}
```

Comment:
An `a` element should never be inside a `button` (or vice versa), so this CSS should never come into play.